packages/context/src/inject-config.ts

```diff
@@ -4,13 +4,27 @@
 // License text available at https://opensource.org/licenses/MIT
 
 import {BindingFilter} from './binding-filter';
-import {BindingKey} from './binding-key';
+import {BindingAddress, BindingKey} from './binding-key';
 import {Context} from './context';
 import {ContextView} from './context-view';
 import {assertTargetType, inject, Injection, InjectionMetadata} from './inject';
 import {ResolutionSession} from './resolution-session';
 import {getDeepProperty, ValueOrPromise} from './value-promise';
 
+/**
+ * Injection metadata for `@config.*`
+ */
+export interface ConfigInjectionMetadata extends InjectionMetadata {
+  /**
+   * Path to retrieve the configuration of the target binding
+   */
+  configPath?: string;
+  /**
+   * Target binding key to override the current binding
```

Comment:
```suggestion
   * Target binding key to override the current binding key
```

Reply:
Let's make this text more descriptive please and explain that this option allows consumers to receive configuration specified for another binding.